pages/ext/discord.vue

```diff
@@ -1,241 +1,188 @@
 <template>
-    <div class="py-10 md:py-24 lg:py-24">
-        <banner>
-            <core-h1 :title="$t('title')"></core-h1>
-            <i18n class="leading-6" tag="p" path="pageAbstract">
-                <template #setup>
-                    <span
-                        class="appendixHint"
-                        @click="getPathWithAppendixAnchor"
-                    >
-                        {{ $t('setupText') }}
-                    </span>
-                </template>
-            </i18n>
-        </banner>
-        <i18n-page-wrapper class="pt-8 px-8 md:px-57 lg:px-56">
-            <div>
-                <div class="section">
-                    <core-h1 :title="$t('tutorialsHeader')"></core-h1>
-                    <p class="paragraphTitle">
-                        {{ $t('registration.title') }}
-                    </p>
-                    <i18n
-                        v-for="(description, i) in $t(
-                            'registration.descriptions',
-                        )"
-                        :key="`discord_registration_${i}`"
-                        :path="`registration.descriptions.${i}`"
-                        class="paragraphContent"
-                        tag="p"
-                    >
-                        <template #br><br /></template>
-                        <template #info>
-                            <span class="category">INFORMATION</span>
-                        </template>
-                        <template #registrationDesk>
-                            <span class="channel">⚠｜registration-desk</span>
-                        </template>
-                        <template #registrationCommand>
-                            <code v-if="!isValidAttendee" class="code"
-                                >!register [TOKEN]</code
-                            >
-                            <span v-else class="code"
-                                >!register {{ token }}</span
-                            >
-                        </template>
-                        <template #general>
-                            <span class="channel">#general</span>
-                        </template>
-                        <template #manager>
-                            <span class="code">discord manager</span>
-                        </template>
-                        <template #mentionManager>
-                            <span class="code">@discord manager</span>
-                        </template>
-                    </i18n>
+    <i18n-page-wrapper class="pt-8 px-8 md:px-57 lg:px-56">
```

Comment:
- 是否打錯，tailwind 應該沒有 57
- md lg 一樣的話，留比較小的那個就好
```suggestion
    <i18n-page-wrapper class="pt-8 px-8 md:px-56">
```

Reply:
這個我好像沒有改過，應該是本來就 57
我在看一下兩個差別

Reply:
真的欸 原本的 code 就是這樣XDD
但 tailwind 應該是只有 px-56 ([reference](https://v2.tailwindcss.com/docs/padding))，再麻煩幫試一下 RWD 會不會有什麼問題 🙏🏽